seed/templates/seed/_sidebar.html

```diff
@@ -1,41 +1,67 @@
 {% load compress %}{# requires menu_controller angularjs controller and menu_controller.js #}
 <div ng-class="{expanded: expanded_controller, collapsed: collapsed_controller, sidebar: true, hide_transition:is_initial_state}">
     <div class="menu">
-        <a id="sidebar-profile" class="item" ui-sref="profile" ui-sref-active="active">
+        <a id="sidebar-toggle" class="item" ng-click="toggle_menu()">
+            <div class="icon" ng-class>
+                <i class="fa fa-bars"></i>
+                <i id="expand-collapse-icon" class="fa" ng-class="{'fa-arrow-right': collapsed_controller, 'fa-arrow-left': expanded_controller}"></i>
+            </div>
+        </a>
+        <div class="divider"></div>
+        <a id="sidebar-profile" class="item" ng-class="{'disabled-item': !logged_in}" ui-sref="profile" ui-sref-active="active">
             <div class="icon"><i class="fa fa-cog"></i></div>
             <div ng-show="!is_initial_state" class="item_name">{$ username $}</div>
         </a>
         <div class="divider"></div>
-        <a id="sidebar-inventory" ng-class="{item: true, active: is_active('/properties') || is_active('/taxlots')}" href="{% url "seed:home" %}#/properties">
-            <div class="icon"><img src="{{ STATIC_URL }}seed/images/icon-city.svg" title="{$:: 'Inventory' | translate $}"></div>
+        <a id="sidebar-inventory" ng-class="{item: true, active: is_active('/properties') || is_active('/taxlots'), 'disabled-item': !logged_in}" href="{% url "seed:home" %}#/properties">
+            <div class="icon">
```

Comment:
had to directly insert svg path here b/c we need to change the fill color using css